operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -319,9 +320,9 @@ function createWLDomain() {
   fi
 
   # copy the filter related files to the wdt lib
-
   cp ${WDT_FILTER_JSON} ${WDT_ROOT}/lib/model_filters.json
   cp ${WDT_CREATE_FILTER} ${WDT_ROOT}/lib
+  cp ${WDT_MII_FILTER} ${WDT_ROOT}/lib
```

Comment:
consider appending `|| trace SEVERE "cp failed" && exitOrLoop` to each of the cp commands (fail-fast)